Library/Formula/i2p.rb

```diff
@@ -0,0 +1,19 @@
+class I2p < Formula
+  desc "I2P is an anonymous overlay network - a network within a network"
+  homepage "https://geti2p.net"
+  url "https://download.i2p2.de/releases/0.9.21/i2pinstall_0.9.21.jar"
+  sha256 "0238ffc6ea44099ef4fe6c20913cb4eec675c2760aea07dfe7d499addcc89cf2"
+
+  depends_on :java => "1.6+"
+
+  def install
+    open("path.conf", "w") do |f|
+      f.write("INSTALL_PATH=#{libexec}")
+    end
+
+    system "java", "-jar", "i2pinstall_#{version}.jar", "-options", "path.conf"
+    libexec.install_symlink libexec/"i2psvc-macosx-universal-64" => "i2psvc"
```

Comment:
Should I use `Hardware::CPU.is_64_bit?` to separate 64-bit wrapper from 32-bit?

This feature seems deprecated according to `homebrew/Library/Homebrew/hardware.rb`:
```ruby
def bits
  64
end
```